plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/TrinoGlueCatalog.java

```diff
@@ -384,6 +385,23 @@ public void registerTable(ConnectorSession session, SchemaTableName schemaTableN
         createTable(schemaTableName.getSchemaName(), tableInput);
     }
 
+    @Override
+    public void unregisterTable(ConnectorSession session, SchemaTableName schemaTableName)
```

Comment:
(can be done as a follow-up as well) : Can we also add a corresponding test into `io.trino.plugin.iceberg.TestIcebergMetastoreAccessOperations` ?

Reply:
I don't think metastore access count is important for `unregister_table` procedure.